example/macos/example/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "example",
+  "version": "0.0.1",
+  "private": true,
+  "scripts": {
+    "android": "react-native run-android",
+    "ios": "react-native run-ios",
+    "start": "react-native start",
+    "test": "jest",
+    "lint": "eslint .",
+    "start:macos": "node node_modules/react-native-macos/local-cli/cli.js start --use-react-native-macos"
+  },
+  "dependencies": {
+    "react": "16.9.0",
+    "react-native": "0.61.5",
```

Comment:
Can we delete this dependency? We shouldn't need react-native if we have react-native-macos

Reply:
no, you do indeed need both.  The CLI and tooling still gets run from react-native.  -- This aligns with other out of tree platforms, such as react-native-windows.

Reply:
Thought that got pulled in separately from react-native-macos. Thanks for the clarification there Andrew